core/src/apps/homescreen/lockscreen.py

```diff
@@ -33,6 +39,33 @@ def render_lock(self) -> None:
         )
         ui.display.icon(45, 202, res.load(ui.ICON_CLICK), ui.TITLE_GREY, ui.BG)
 
+    def handle_rendering(self) -> loop.Task:  # type: ignore
+        """Task that is rendering the layout in a busy loop.
+
+        Copy-pasted from ui.Layout.handle_rendering with modification to set the
+        backlight to a lower level while lockscreen is on, and a longer sleep because
+        we never do any redrawing."""
+        # Before the first render, we dim the display.
+        ui.backlight_fade(ui.BACKLIGHT_DIM)
+        # Clear the screen of any leftovers, make sure everything is marked for
+        # repaint (we can be running the same layout instance multiple times)
+        # and paint it.
+        ui.display.clear()
+        self.on_render()
+
+        if __debug__ and self.should_notify_layout_change:
+            from apps.debug import notify_layout_change
+
+            self.should_notify_layout_change = False
+            notify_layout_change(self)
```

Comment:
That doesn't really seem worth it? This is a one-off. If we end up doing something like this in more places, the requirements will become clearer.

(fwiw if we want to get rid of this, we could change `backlight_fade(style.BACKLIGHT_NORMAL)` to `backlight.fade(self.backlight_level)` and remove the copy-paste. It didn't seem worth polluting all Layouts with a new variable, but maybe the cost is negligible.
same thing with the `loop.sleep()` which could also be a class or instance variable, and in case of homescreen and lockscreen could be changed to something like `loop.DO_NOT_RESCHEDULE`)

Reply:
> backlight.fade(self.backlight_level)

I would do it. My concern is that if we change something in `Layout.handle_rendering` we forget to add it here. IMHO.

Reply:
done in 7a2b8e748109165d649d3a01f71362519172de20